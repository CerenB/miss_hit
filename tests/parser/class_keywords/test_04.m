% (c) Copyright 2019 Florian Schanda

function r = test_04
  l = leek;
  properties(l)
end
