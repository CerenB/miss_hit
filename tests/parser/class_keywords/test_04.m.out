----------------------------------------------------------------------
--  Parse tree for test_04.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <test_04>
      Outputs: Identifier <r>
    Body: Sequence_Of_Statements
      Statements: Simple_Assignment_Statement
        LHS: Identifier <l>
        RHS: Identifier <leek>
      Statements: Naked_Expression_Statement
        Expression: Reference
          Name: Identifier <properties>
          Arguments: Identifier <l>
----------------------------------------------------------------------
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
